extensions/python/pythonprocessors/examples/CountingProcessor.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+global i
+i = 0
+
+
+class WriteReverseStringCallback:
```

Comment:
what makes this "reverse"

Reply:
A copy-paste error :sweat_smile: , fixed it in https://github.com/apache/nifi-minifi-cpp/pull/1504/commits/3d22eca3cf03cff48d06247a6e4f0ce54eedfb81